components/TextFields/src/MDCTextInput.h

```diff
@@ -201,6 +201,12 @@ typedef NS_ENUM(NSUInteger, MDCTextInputTextInsetsMode) {
 /** The underline view */
 @property(nonatomic, nullable, strong, readonly) MDCTextInputUnderlineView *underline;
 
+/** A boolean value that indicates whether the text input contains content **/
```

Comment:
If this goes back into TextFields, please break these changes out into separate PRs. This broke internally and doesn't compile at all with bazel.